packages/sdk/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "@pipcook/sdk",
+  "version": "1.0.0",
+  "description": "JavaScript SDK for accessing Pipcook Daemon",
+  "main": "dist/index.js",
+  "scripts": {
+    "build": "npm run clean && npm run compile",
+    "clean": "rm -rf ./dist && rm -rf tsconfig.tsbuildinfo",
+    "compile": "tsc -b tsconfig.json"
```

Comment:
Because we are using tsc in this script, so the `typescript` should be in `devDependencies`.